sigar_windows.go

```diff
@@ -41,11 +42,61 @@ func (self *Swap) Get() error {
 }
 
 func (self *Cpu) Get() error {
-	return notImplemented()
+	cpuQueries := []string{
+		`\processor(_Total)\% idle time`,
+		`\processor(_Total)\% user time`,
+		`\processor(_Total)\% privileged time`,
+		`\processor(_Total)\% interrupt time`,
+	}
+	queryResults, err := runRawPdhQueries(cpuQueries)
+	if err != nil {
+		return err
+	}
+
+	self.populateFromPdh(queryResults)
+	return nil
 }
 
 func (self *CpuList) Get() error {
-	return notImplemented()
+	cpuQueries := []string{
+		`\processor(*)\% idle time`,
+		`\processor(*)\% user time`,
+		`\processor(*)\% privileged time`,
+		`\processor(*)\% interrupt time`,
+	}
+	// Run a PDH query for all CPU metrics
+	queryResults, err := runRawPdhArrayQueries(cpuQueries)
+	if err != nil {
+		return err
+	}
+	self.List = make([]Cpu, 0)
+	for cpu, counters := range queryResults {
+		index := 0
+		if cpu == "_Total" {
```

Comment:
I don't suppose that query is a fully featured regex and we could do `\processor([0-9]+)\% idle time`?